rename_images_by_date.py

```diff
@@ -42,68 +44,123 @@ def findDuplicates(file_dict):
 			
 	return (duplicates, file_dict)
 
-if len(sys.argv) < 2:
-	print("[error] dir name not provided")
-	exit(-1)
+# Create the argument parser, that will display "rename" as program name in output
+my_parser = argparse.ArgumentParser(prog="rename", description='Rename file following given pattern.')
 
+# Add the arguments to parse in the CLI
+#TODO make real alternative option if necessary
+my_parser.add_argument('Dir',
+                       metavar='pathTargetDirectory',
+                       type=str,
+                       help='Idk. Surely target folder to rename pics')
+                     
+my_parser.add_argument('--undated',
+                        action='store_true',
+                        help='If specified, pictures do not need date for renaming')
+my_parser.add_argument('--cam','-c',
+                       metavar='cameraName',
+                       type=str,
+                       help='Idk. Maybe the camera name to use for a renaming')
+my_parser.add_argument('--time','-t',
+                       metavar='timeDelay',
+                       type=str,
+                       help='Idk. Something about time')
+my_parser.add_argument('--suffix', '-s',
+                      metavar='suffix',
+                      type=str,
+                      default="",
+                      help='Specify suffix for new renaming')
+my_parser.add_argument('--prefix', '-p',
+                      metavar='prefix',
+                      type=str,
+                      default="",
+                      help='Specify prefix for new renaming')                       
+my_parser.add_argument('--verbose', '-v',
+                        action='store_true',
+                        help='If specified, plenty of printings')
+# Execute the parse_args() method
+args = my_parser.parse_args()
+                       
 number=1
-cam_name = ''
+cam_name = args.cam
+time = args.time
```

Comment:
well. You can see that the variable "time" got its value from the user input while "time_delay" is of Date type. But then, if you mean to rename the variable, I agree that is confusing. Renamed variables to avoid confusion